applications/callflow/src/cf_exe.erl

```diff
@@ -595,7 +644,6 @@ launch_cf_module(#state{flow=?EMPTY_JSON_OBJECT}=State) ->
 launch_cf_module(#state{call=Call
                         ,flow=Flow
                        }=State) ->
-    lager:debug("launching next flow module: ~p", [Flow]),
```

Comment:
@fenollp because it flow is a large json object and this is not logging something specific.  Rather than fill the logs with erlang data objects we try to create logging that describes what is occurring.